frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { StudioTableLocalPagination } from './StudioTableLocalPagination';
+import { columns, rows } from '../StudioTableRemotePagination/mockData';
+
+describe('StudioTableLocalPagination', () => {
+  const paginationProps = {
```

Comment:
Here we can use typescript to ensure type safety. 😊 
```suggestion
  const paginationProps: StudioTableLocalPagination["pagination"] = {
```

Reply:
Fixed 😊